CONTRIBUTING.md

```diff
@@ -1,113 +1,29 @@
-# Contributing #
+Contributing
+============
 
-Information on contributing to this repo is in the [Contributing Guide](https://github.com/aspnet/Home/blob/dev/CONTRIBUTING.md) in the Home repo.
+The document covers the basic process for contributing to the [ASP.NET Core documentation site](https://docs.asp.net). See the [ASP.NET Core contributing guide](https://github.com/aspnet/Home/blob/dev/CONTRIBUTING.md) for information on contributing to ASP.NET Core source.
```

Comment:
Second sentence would be clearer with "for info on..." first, followed by "see ..."